core/domain/feedback_domain.py

```diff
@@ -41,12 +41,24 @@ class FeedbackThread(python_utils.OBJECT):
             created.
         last_updated: datetime.datetime. The time when the feedback thread
             was last updated.
+        last_message_id: int. The id of the last message in the thread.
```

Comment:
This and other args should explain when the value is None. (If you look carefully at the logic, there are cases when it might be None.)